terraso_backend/apps/soil_id/models/depth_dependent_soil_data.py

```diff
@@ -23,3 +23,16 @@ class DepthDependentSoilData(BaseModel):
     soil_data_input = models.ForeignKey(SoilData, on_delete=models.CASCADE)
     depth_top = models.IntegerField(blank=True)
     depth_bottom = models.IntegerField(blank=True)
+    slope = models.IntegerField(blank=True)
+    texture = models.CharField(blank=True)
+    color_hue_float = models.FloatField(blank=True)
+    color_hue_char = models.CharField(blank=True)
+    color_value = models.FloatField(blank=True)
+    color_chroma = models.IntegerField(blank=True)
```

Comment:
for color:
- the wikipedia for munsell colors refers to the numeric part of hues as `substeps`, so maybe we can go with `color_hue` and `color_hue_substep`? https://en.wikipedia.org/wiki/Munsell_color_system#Hue
- same again with choices, and i think we should make them all char fields (whose values might be a string representation of a number) because these aren't actually numbers we would ever want to add together